jetstream/external_config.py

```diff
@@ -37,32 +36,29 @@ class ExternalConfigCollection:
 
     configs: List[ExternalConfig] = attr.Factory(list)
 
-    JETSTREAM_CONFIG_REPO = "mozilla/jetstream-config"
+    JETSTREAM_CONFIG_URL = "https://github.com/mozilla/jetstream-config"
 
     @classmethod
     def from_github_repo(cls) -> "ExternalConfigCollection":
         """Pull in external config files."""
+        # download files to tmp directory
+        with TemporaryDirectory() as tmp_dir:
+            repo = Repo.clone_from(cls.JETSTREAM_CONFIG_URL, tmp_dir)
 
-        g = Github()
-        repo = g.get_repo(cls.JETSTREAM_CONFIG_REPO)
-        files = repo.get_contents("")
+            external_configs = []
 
-        if isinstance(files, ContentFile):
-            files = [files]
+            for config_file in tmp_dir.glob("**/*.toml"):
+                last_modified = list(repo.iter_commits("main", paths=config_file))[0].committed_date
```

Comment:
we can permit lazy evaluation here

```suggestion
                last_modified = next(repo.iter_commits("main", paths=config_file)).committed_date
```